app/modules/widgets/utils/analytics-events.js

```diff
@@ -0,0 +1,52 @@
+import ReactGA from 'react-ga'
+
+class AnalyticsEvents {
```

Comment:
Maybe this file does not necessarily have to be a class, but I also do not see problems in being... Besides, I think it was very good ... Each method with its responsibility well described.